controllers/v1/library/observationController.js

```diff
@@ -0,0 +1,256 @@
+/**
+ * name : observationController.js
+ * author : Akash
+ * created-date : 04-Jun-2020
+ * Description : Observation solution library related information.
+ */
+
+const observationHelper = require(MODULES_BASE_PATH + "/library/observation/helper");
+
+ /**
+    * ObservationLibrary
+    * @class
+*/
+module.exports = class ObservationLibrary {
+    
+    constructor() {}
+
+    static get name() {
+        return "ObservationLibrary";
+    }
+
+    /**
+    * @api {get} /assessment/api/v1/library/observation/list List of observation library
+    * @apiVersion 1.0.0
+    * @apiName List of observation library
+    * @apiGroup Observation Library
+    * @apiSampleRequest /assessment/api/v1/library/observation/list
+    * @apiUse successBody
+    * @apiUse errorBody
+    * @apiParamExample {json} Response:
+    * {
+    * "message": "Template solutions fetched successfully",
```

Comment:
@aman-tunerlabs - Please send message as per what is returned like - "Observation solution templates fetched successfully."